runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -94,10 +95,19 @@
 @PluginDescriptor(
 	name = "Timers",
 	description = "Show various timers in an infobox",
-	tags = {"combat", "items", "magic", "potions", "prayer", "overlay"}
+	tags = {"combat", "items", "magic", "potions", "prayer", "overlay", "abyssal", "sire"}
 )
 public class TimersPlugin extends Plugin
 {
+	private static final int ABYSSAL_SIRE_STUNNED_NPC_ID = NpcID.ABYSSAL_SIRE_5887;
```

Comment:
Just use the NpcID class directly no need to store the IDs as constants here

Reply:
I was trying to make it more readable since NpcID doesn't explicitly say which state the id represents.

Reply:
Comment is enough, the constants were not very helpful as well.